workspace_tools/export/gcc_arm_common.tmpl

```diff
@@ -1,12 +1,38 @@
 # This file was automagically generated by mbed.org. For more information, 
 # see http://mbed.org/handbook/Exporting-to-GCC-ARM-Embedded
 
+ifeq ($(OS),Windows_NT)
+MKDIR := mkdir
+else
+MKDIR := mkdir -p
+endif
+
+ifeq (,$(filter bin,$(notdir $(CURDIR))))
+.SUFFIXES:
+OBJDIR := bin
+MAKETARGET = $(MAKE) --no-print-directory -C $@ -f $(CURDIR)/Makefile \
+		SRCDIR=$(CURDIR) $(MAKECMDGOALS)
+.PHONY: $(OBJDIR) clean
+$(OBJDIR):
+	+@[ -d $@ ] || $(MKDIR) $@
+	+@$(MAKETARGET)
+Makefile : ;
+% :: $(OBJDIR) ; :
+clean :
+	rm -rf $(OBJDIR)
```

Comment:
@akhilpanayamparambil Thanks. we need to assume also windows only. I'll give it a run in the cmd windows